### DISCUSSION TITLE
Turning on the PC works well, but an error occurs when shutting down.

### DISCUSSION THREAD

**Discussion by yksoon** [AlexGustafsson/homebridge-wol#169] [Questions] [answered]
I don't use raspberry pi, I use synology nas. Turning on the PC works well, but an error occurs when shutting down. I can't use `apt-get` or `yum` because I'm using synology. What should I do? I've seen you install `sudo apt-get install samba-common-bin` before.

```
[8/20/2021, 12:36:05 AM] [My PC] An error occured while trying to shut down "My PC" (192.168.0.7): Error: Command failed: net rpc shutdown --ipaddress 192.168.0.7 --user username%password
/bin/sh: net: not found

    at ChildProcess.exithandler (child_process.js:390:12)
    at ChildProcess.emit (events.js:400:28)
    at maybeClose (internal/child_process.js:1055:16)
    at Socket.<anonymous> (internal/child_process.js:441:11)
    at Socket.emit (events.js:400:28)
    at Pipe.<anonymous> (net.js:675:12) {
  killed: false,
  code: 127,
  signal: null,
  cmd: 'net rpc shutdown --ipaddress 192.168.0.7 --user username%password',
  stdout: '',
  stderr: '/bin/sh: net: not found\n'
}
```

**Comment by AlexGustafsson** [comment, marked as answer]
Try to ask your question on the super user stack exchange, they may be able to help you. I think most people using homebridge on a Synology NAS runs it using Docker.

**Reply by yksoon** [reply]
How can I exchange the super user stack?

**Reply by AlexGustafsson** [reply]
It’s a website for asking questions others can answer. You can visit it here: https://superuser.com/

**Reply by yksoon** [reply]
Thanks friend. 
I found the answer there. I added and restarted the container at startup.sh in Homebridge as follows. That's how it all works.

`apk add --no-cache samba samba-common-tools`
`apk add --no-cache openssh-client`

**Reply by AlexGustafsson** [reply]
Great! Thanks for sharing the solution.
